test/truffle/memory/minimum-heap.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+# 
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+# Calculates the approximate minimum heap sized needed to run hello world.
+# Not run it fast - just run it at all.
+
+# For example:
+# $ ruby test/truffle/memory/minimum-heap.rb bin/jruby "-X+T -e 'puts 14'"
+
+tolerance = 5
+max_iterations = 100
+
+lower = 0
+upper = 4 * 1024
```

Comment:
It was just an initial size that was bound to be big enough. But now I think about it I can also search for the upper bound and not have one hard coded. I'll do that.